experimenter/experimenter/targeting/constants.py

```diff
@@ -2270,6 +2270,23 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+NO_HTTPS_ONLY = NimbusTargetingConfig(
```

Comment:
You might want to rename this to `NO_HTTPS_ONLY_DESKTOP` since youre adding a separate targeting for Fenix.